test/fixtures/users.yml

```diff
@@ -4,8 +4,12 @@
 # model remove the '{}' from the fixture names and add the columns immediately
 # below each fixture, per the syntax in the comments below
 #
-one: {}
+one: {
+    email: example@example.com
```

Comment:
This is weird, I've never seen the `{}` syntax in yml before. I guess if it works I'm down. This has weird spacing compared to line 13 though.

Reply:
No problem.  I'll update to the more expected yml syntax.